test/remote/gateways/remote_authorize_net_cim_test.rb

```diff
@@ -179,7 +198,7 @@ def test_successful_create_customer_payment_profile_request
 
     assert response.test?
     assert_success response
-    assert_nil response.authorization
```

Comment:
This is no longer nil on any of these calls, it matches with the customer profile id.